linera-witty/tests/wit_load.rs

```diff
@@ -353,6 +354,52 @@ fn test_heap_allocated_fields() {
     );
 }
 
+/// Check that a [`Vec`] type is properly loaded from memory and lifted from its flat
+/// layout.
+#[test]
+fn test_vec() {
+    let expected = vec![SimpleWrapper(false), SimpleWrapper(true)];
+
+    test_load_from_memory(&[8, 0, 0, 0, 2, 0, 0, 0, 0, 1], expected.clone());
+    test_lift_from_flat_layout(hlist![0_i32, 2_i32], expected, &[0, 1]);
```

Comment:
(So the `input` here is: "`2` bytes at address `0`". And those two bytes are `0` for `false` and `1` for `true`?)

Reply:
Exactly :raised_hands: